_forms/three-centers.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: form
+title: Three Centers
+title_mark: Centers
+published_date: 2025-11-09
+refactored_date: 2025-11-09
+authors:
+  - kyle-ingersoll
+  - michael-basil
+excerpt: "Three Centers aligns your Head, Heart, and Hara so that they communicate properly."
+---
+
+## Intention
+
+Three Centers aligns your Head, Heart, and Hara so that they communicate properly. It acts as sort of a gut check, and can be done sitting down, standing, or laying on the floor.
+
+## Experience
+
+You relax completely, feeling the Ki (energy flow) through the Three Centers. Through alignment of those centers, you feel the inner turmoil drain away, and a sense of purpose rises within you.
+
+## Implementation
+
+You make sure your big toes are touching something, timing your movement with the exhale. You have your vision at 180 degrees, and then you do Hara breathing (Belly breathing). Feel the Heart and Hara align with your hands, and once you have a feeling of how they work together while breathing, shift your hands, palms facing forward, to the sides of your head so you can just barely see them on the edge of your 180 degree vision. Once all Three Centers align, then you can go about your day.
```

Comment:
The last sentence .. I would suggest something that once all three centers align, you will feel clearer inviting discernment.   Maybe not this exactly, but something to suggest how that alignment influence what was stucking, gapping or misaligned.   

Something than answers ...

How has my state or condition been influenced as I go about my day?